src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerTestingController.java

```diff
@@ -97,6 +102,43 @@ public CaseEventMessage getMessagesByMessageId(@PathVariable("message_id") final
         }
     }
 
+    @ApiOperation("Gets the case event message by messageId")
+    @ApiResponses({
+        @ApiResponse(
+            code = 200,
+            message = "Messages returned successfully",
+            response = CaseEventMessageEntity.class)
+    })
+    @GetMapping("/messages/query")
+    @SuppressWarnings("PMD.UseObjectForClearerAPI")
+    public EventMessageQueryResponse getMessagesByQueryParameters(
+        @RequestParam(value = "states", required = false) final String states,
+        @RequestParam(value = "case_id", required = false) final String caseId,
+        @RequestParam(value = "event_timestamp", required = false) final String eventTimestamp,
+        @RequestParam(value = "from_dlq", required = false) final Boolean fromDlq) {
+
+        if (isNonProdEnvironment()) {
+            return eventMessageQueryService.getMessages(states, caseId, eventTimestamp, fromDlq);
+        } else {
+            throw new CaseEventMessageNoAllowedRequestException();
+        }
+    }
+
+    @ApiOperation("Deletes the case event message by messageId")
+    @ApiResponses({
+        @ApiResponse(
+            code = 200,
+            message = "Messages deleted successfully")
+    })
+    @DeleteMapping("/messages/{message_id}")
+    public void deleteMessageByMessageId(@PathVariable("message_id") final String messageId) {
+        if (isNonProdEnvironment()) {
+            eventMessageReceiverService.deleteMessage(messageId);
+        } else {
+            throw new CaseEventMessageNoAllowedRequestException();
```

Comment:
typo ? missing  `t` from `Not`

 CaseEventMessageNoAllowedRequestException

Reply:
done